src/robot/utils/typehints.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2008-2015 Nokia Networks
+#  Copyright 2016-     Robot Framework Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any, Callable, TypeVar, Generic
+
+
+T = TypeVar('T', bound=Callable[..., Any])
+
+# Type Alias for objects that are only known at runtime. This should be Used as a
+# default value for generic classes that also use `@copy_signature` decorator
+known_at_runtime = type(object)
```

Comment:
- As I commented elsewhere, types should be named like `KnownAtRuntime` or `TypeType`.
- I'm not certain is there a need to have this in utils, it could just be created where needed and in some cases `type(object)` could be enough without a named type alias. This is especially true if in some cases we can use more detailed type than `type(object)` with `copy_signature`.

Reply:
Updated the Name. My follow-up PR for `Body` will use this value, I can define it a second time there, but it seems redundant.